micro_rpc/proto/messages.proto

```diff
@@ -16,7 +16,7 @@
 
 syntax = "proto3";
 
-package oak.protobuf_idl;
+package micro_rpc;
```

Comment:
Do we need an `oak.` in front?

Reply:
I'm trying to make it agnostic, but we'll see.